pkg/kapp/diff/change.go

```diff
@@ -59,8 +63,11 @@ func NewChange(existingRes, newRes, appliedRes ctlres.Resource) *ChangeImpl {
 	if appliedRes != nil {
 		appliedRes = appliedRes.DeepCopy()
 	}
+	if clusterOriginalRes != nil {
+		clusterOriginalRes = clusterOriginalRes.DeepCopy()
```

Comment:
The motivation here is that the original resource should not be changed.
This might be unnecessary here as we do not really modify `ClusterOriginalResource` but that said it does not hurt and I am fine with having this as a safeguard to prevent future changes from breaking things.
(No action item here, just tracking thoughts - this comment can be resolved)